plugins/processors/awsneuron/config.go

```diff
@@ -0,0 +1,19 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package awsneuron
+
+import (
+	"go.opentelemetry.io/collector/component"
+)
+
+type Config struct{}
+
+// Verify Config implements Processor interface.
+var _ component.Config = (*Config)(nil)
```

Comment:
why are we doing this?

Reply:
not relevant now, merged neuron processor with gpu